src/main/java/edu/hawaii/its/groupings/service/OotbActiveUserProfileService.java

```diff
@@ -0,0 +1,60 @@
+package edu.hawaii.its.groupings.service;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.security.core.authority.AuthorityUtils;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+
+import edu.hawaii.its.groupings.access.User;
+
+public class OotbActiveUserProfileService implements UserDetailsService {
+    private final Map<String, User> users = new HashMap<>();
+
+    public OotbActiveUserProfileService() {
+        initUsers();
+    }
+
+    private void initUsers() {
+        // Initialize user with UID "user0123"
+        users.put("USER", new User.Builder("user0123")  // UID is clearly the first parameter in the constructor
+                .uhUuid("11111111")
+                .authorities(AuthorityUtils.createAuthorityList("ROLE_UH", "ROLE_OOTB"))
+                .addAttribute("cn", "user")
+                .addAttribute("mail", "user@hawaii.edu")
+                .addAttribute("givenName", "DefaultUser")
+                .build());
+
+        // Initialize owner with UID "owner0123"
+        users.put("OWNER", new User.Builder("owner0123")
+                .uhUuid("22222222")
+                .authorities(AuthorityUtils.createAuthorityList("ROLE_UH", "ROLE_OWNER", "ROLE_OOTB"))
+                .addAttribute("cn", "owner")
+                .addAttribute("mail", "owner@hawaii.edu")
+                .addAttribute("givenName", "OwnerUser")
+                .build());
+
+        // Initialize admin with UID "admin0123"
```

Comment:
We tend to always reference uid as lowercase rather than UID. Change the uppercase UID to lowercase, thanks!